src/tools/populateLegacyResourceTitlesCli.js

```diff
@@ -0,0 +1,28 @@
+import readline from 'readline';
+import moment from 'moment';
+import processLegacyResources from './populateLegacyResourceTitles';
+import { DATE_FORMAT } from '../constants';
+import { auditLogger } from '../logger';
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+rl.question('Please enter a START date in the format MM/DD/YYYY (ENTER for -30 days):', async (startDate) => {
+  const start = !startDate ? moment().subtract(30, 'd') : moment(startDate);
```

Comment:
```suggestion
  const start = !startDate ? moment().subtract(30, 'd') : moment(startDate, 'MM/DD/YYYY');
```

Reply:
This has been fixed along with the sequelize issue.